packages/analyticsdx-template-lint/src/schemas/variables-schema.json

```diff
@@ -175,7 +206,7 @@
               "items": {
                 "type": ["string", "null"]
               },
-              "doNotSuggest": true
+              "doNotSuggest": false
```

Comment:
This makes `enumsLabels` also show in the code-completion drop-down, since we have the description hover text about the version as well.